app/styles/_tables.less

```diff
@@ -143,6 +143,19 @@
         }
       }
     }
+    // remove extra 2px border on <tr> in nested ng-repeats.
+    // since we cannot select the "first element of parent with class ..."
+    // select every <tr> with the .nested-loop class, apply a 0px border-top-width,
+    // and then re-apply the 2px border-top-width to every tr.nested-loop child
+    // except the first one to avoid tables with 4px border-top-widths.
+    > tbody.nested-loop-container {
+      > tr.nested-loop {
+        border-top-width: 0;
+      }
+      > tr.nested-loop ~ tr.nested-loop {
+          border-top-width: 2px;
```

Comment:
I guess I'm not clear on why there is a top border on the first hidden row. If `display: none`, shouldn't the border be hidden as well?